internal/e2e/testdata/rank_stats.txt

```diff
@@ -1,4 +1,4 @@
 queries: 16
-recall@1: 8 (50%)
+recall@1: 9 (56%)
 recall@5: 11 (69%)
-mrr: 0.600787
+mrr: 0.632037
```

Comment:
Tiny improvement, because "sourcegraph/conc" is now ranked higher than a ~2 months older "golang/go"